src/ios/CordovaIntercom.m

```diff
@@ -88,4 +88,12 @@ - (void) checkForUnreadMessages:(CDVInvokedUrlCommand*)command
     [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
 }
 
+- (void) setDeviceToken:(CDVInvokedUrlCommand*)command
+{
+    NSString* deviceToken = [command.arguments objectAtIndex:0];
+    NSData* data = [deviceToken dataUsingEncoding:NSUTF8StringEncoding];
+    [Intercom setDeviceToken:data];
+
```

Comment:
nit: remove empty line.

Also you need to add a plugin result here (see the methods above).